### ISSUE TITLE
Quality regression with Nerfacto in v0.1.19

### ISSUE THREAD

**Issue by fangchangma** [nerfstudio-project/nerfstudio#1670]
**Describe the bug**
Quality regression with Nerfacto in v0.1.19, across various datasets, with loss of sharpness and detail.

**To Reproduce**
```
# within the local git repo

# good baseline
git checkout v0.1.18
python scripts/train.py nerfacto --vis=wandb nerfstudio-data --data ../data/nerfstudio/Egypt/

# problematic version
git checkout v0.1.19
python scripts/train.py nerfacto --vis=wandb nerfstudio-data --data ../data/nerfstudio/Egypt/
```

**Expected behavior**
The two versions should have similar level of quality. 

**Screenshots**
Below are validation images at step 28500 (left is ground truth, and right is the rendering). The gap is more visible in the rendered videos.
v0.1.18
![image](https://user-images.githubusercontent.com/17073991/228133089-4f54a893-7867-4f6a-8784-250afdcdbc3d.png)

v0.1.19
![image](https://user-images.githubusercontent.com/17073991/228133052-821337f2-85ce-4e62-bdb4-f0ef020c1643.png)

Comparison of evaluation loss over the coarse of training (grey is v0.1.18, purple is v0.1.19)
![W B Chart 3_27_2023, 10_06_07 PM](https://user-images.githubusercontent.com/17073991/228133881-99176951-b2a0-466c-bc85-a57dffed62f0.png)

**Comment by chris838** [comment]
hmm, just tried on Colab and I can't reproduce - I get the higher PSNR for both versions, at least for the first 5k iterations

**Comment by chris838** [comment]
<img width="1434" alt="image" src="https://user-images.githubusercontent.com/925254/228199292-9af028c9-7ac8-42d3-bfac-fa4d5880cf0f.png">

**Comment by chris838** [comment]
any more details on the platform you're using?

**Comment by chris838** [comment]
FWIW I ran the same test with camera optimisation off, just in case this was the culprit (before I realised I couldn't actually reproduce the issue) and also get the same results for both versions.

<img width="1427" alt="image" src="https://user-images.githubusercontent.com/925254/228201050-f42203ce-6db4-4ecb-a03f-35b8254c1ce2.png">

**Comment by fangchangma** [comment]
> any more details on the platform you're using?

My colleague and I were both able to reproduce the problem on our two different Ubuntu machines, one with 3080 and the other with 4090. For the 4090 machine, I got `cuda_11.7.r11.7`, `torch==1.13.1+cu117`, and `Ubuntu 22.04.2 LTS`


> just tried on Colab and I can't reproduce - I get the higher PSNR for both versions, at least for the first 5k iterations

I also tried running both versions on Colab as you did, but with the following commands. Surprisingly, both v0.1.18 and v0.1.19 produced blurry results on Colab. The PSNR numbers could have been misleading. Would be great to check out the rendered images instead
```
!git checkout v0.1.18
!python scripts/train.py nerfacto \
  --vis=wandb \
nerfstudio-data --data ../data/nerfstudio/Egypt/

# problematic version
!git checkout v0.1.19
!python scripts/train.py nerfacto \
  --vis=wandb \
nerfstudio-data --data ../data/nerfstudio/Egypt/
```

**Comment by fangchangma** [comment]
> just tried on Colab and I can't reproduce - I get the higher PSNR for both versions, at least for the first 5k iterations

I also tried running both versions on Colab as you did, but with the following commands instead
```
!git checkout v0.1.18
!python scripts/train.py nerfacto \
  --vis=wandb \
nerfstudio-data --data ../data/nerfstudio/Egypt/

# problematic version
!git checkout v0.1.19
!python scripts/train.py nerfacto \
  --vis=wandb \
nerfstudio-data --data ../data/nerfstudio/Egypt/
```
Surprisingly, on Colab both versions produced blurry results. Note that the PSNR numbers could have been misleading - please check out the rendered validation images for comparison

**Comment by fangchangma** [comment]
Did a binary search over the commits between v0.1.18 and v0.1.19, and the culprit was commit [0d336a181e76588f3b3cb6ba6419d7efc2bcf297](https://github.com/nerfstudio-project/nerfstudio/commit/0d336a181e76588f3b3cb6ba6419d7efc2bcf297)

**Comment by machenmusik** [comment]
(Is it the change to lr_final that makes the difference? Or maybe max_steps having one less zero?)

**Comment by tancik** [comment]
Those would have caused differences. The current scheduler in master is different now,
`ExponentialDecaySchedulerConfig(lr_final=6e-6, max_steps=200000),`
It was changed to make the optimization more stable. Seem like it could use more hyperparameter tuning.

**Comment by hrh-bbc-rd** [comment]
I'm having the same issue. Top is a screenshot from v0.1.18, bottom is v0.1.19. Same image set and camera path.
<img width="1279" alt="Screenshot 2023-04-06 at 4 26 22 pm" src="https://user-images.githubusercontent.com/113361350/230425877-5ac31cce-bf06-48ae-9547-48541035e5ca.png">

**Comment by tancik** [comment]
Closing as we are now at v0.3.1. Feel free to create a new issue if there are lingering problems.
